orchestra/static/orchestra/common/js/orchestra_api.js

```diff
@@ -0,0 +1,62 @@
+var serviceModule =  angular.module('orchestra.common.services');
+
+serviceModule.factory('orchestraApi', ['$http', function($http) {
+  var actionService = new function() {
+    var actionService = this;
```

Comment:
sanity check: do the other `/orchestra/api/interface/` views (the ones that existed before this PR) also get served from this service?